physx.js

```diff
@@ -511,14 +511,17 @@ const physxWorker = (() => {
     const quaternions = scratchStack.f32.subarray(index, index + maxNumUpdates*4);
     index += maxNumUpdates*4;
     const scales = scratchStack.f32.subarray(index, index + maxNumUpdates*3);
-    index += maxNumUpdates*7;
+    index += maxNumUpdates*3;

```

Comment:
This is a good bugfix, I pushed it upstream: https://github.com/webaverse/app/commit/1f1b392d462ea86436b3c458e79303f1558a6705